modules/UI/authentication/RoomLocker.js

```diff
@@ -1,34 +1,37 @@
-/* global APP, JitsiMeetJS */
```

Comment:
What do you mean? Move all RoomLocker module to Invite one or separate UI part from logic and move it to Invite module?